config/dev/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: system
```

Comment:
Namespace was missing as I removed it form the `giantswarm` overlay while introducing `helm`